services/hierarchy/src/test/kotlin/OrtRunServiceTest.kt

```diff
@@ -94,15 +107,85 @@ class OrtRunServiceTest : WordSpec() {
                 results.totalCount shouldBe 0
             }
         }
+
+        "deleteOrtRun" should {
+            "delete an ORT run" {
+                val ortRunId = createOrtRun()
+                val jobId = createReporterJob(ortRunId)
+
+                val mockReportStorageService = mockk<ReportStorageService> {
+                    coEvery { deleteReport(any() as Long, any() as String) } just Runs
+                }
+
+                val service = createService(mockReportStorageService)
+                service.deleteOrtRun(ortRunId)
+
+                coVerify(exactly = 3) { mockReportStorageService.deleteReport(any() as Long, any() as String) }
+
+                fixtures.reporterRunRepository.getByJobId(jobId) shouldBe null
+                fixtures.ortRunRepository.get(ortRunId) shouldBe null
+            }
+
+            "delete an ORT run even if some reports are not found in storage" {
+                val ortRunId = createOrtRun()
+                val jobId = createReporterJob(ortRunId)
+
+                val mockReportStorageService = mockk<ReportStorageService> {
```

Comment:
In this case, it would be good to check whether other reports are deleted after an exception occured.